client/src/components/styles/hoverStyles.js

```diff
@@ -0,0 +1,116 @@
+import styled, { keyframes } from 'styled-components';
+
+const StyledContent = styled.div`
+padding: 16px;
+margin: 0;
+color: #30455a;
+font-size: 12px;
+display: block;
+`;
+
+const HoverBox = styled.div`
+margin-top: 8px;
+margin-bottom: 8px;
+background: linear-gradient(to bottom, rgba(227,234,239,1) 0%,rgba(199,213,224,1) 100%);
+width: 306px;
+color: #30455a;
+font-size: 12px;
+overflow: hidden;
+box-shadow: 0 0 12px #000000;
+padding: 0;
+margin: 0;
+display: block;
+`;
+
+const StyledScreenshots = styled.div`
+position: relative;
+width: 274px;
+height: 153px;
+margin: 5px 0;
+`;
+
+const ScreenshotHoverFadein = keyframes`
+from {
+  opacity: 0;
+}
+to {
+  opacity: 1;
+}
+`;
+
+const Screenshot = styled.div`
+position: absolute;
+width: 100%;
+height: 100%;
+background-size: cover;
+background-position: center center;
+opacity: 0;
+transition: opacity 300ms;
+animation: ${ScreenshotHoverFadein} 3s linear;
+animation-iteration-count: infinite;
+display: block;
+`;
+
+const StyledReview = styled.span`
+color: ${props => props.summary};
+`;
+
+const StyledTitle = styled.h4`
+color: #222d3d;
+font-size: 15px;
+font-family: "Motiva Sans", Sans-serif;
```

Comment:
Motiva Sans is a paid font and not part of native web clients (as least in my clients in Windows). This might be why the font showed up as default serif in the image that I posted in the comment before this review. I mentioned Google Fonts' Roboto as a similar, free alternative in Slack previously. Recommend switching